index.js

```diff
@@ -34,6 +39,22 @@ var cli = new Liftoff({
   completions: completion,
   extensions: interpret.jsVariants,
   v8flags: v8flags,
+  configFiles: {
+    '.gulprc': {
+      home: {
+        path: '~',
+        extensions: interpret.extensions,
+      },
+      project: {
+        path: '.',
+        extensions: interpret.extensions,
+      },
+      cwd: {
+        path: process.env.INIT_CWD,
+        extensions: interpret.extensions,
```

Comment:
If we support `cwd`, I'm not sure it should be using `INIT_CWD` because someone could specify `--cwd` option that changes the `cwd`.  Thoughts?